core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/models/meta-tags-model.ts

```diff
@@ -148,3 +151,32 @@ export const SEO_TAGS = [
     SEO_OPTIONS.TWITTER_DESCRIPTION,
     SEO_OPTIONS.TWITTER_IMAGE
 ];
+
+export const socialMediaTiles = {
```

Comment:
you can set type of this as `Record<SEO_MEDIA_TYPES, SocialMediaOption` that way if we add another media type, the linter will tell us that this object is incomplete and it will not compile.